### PR TITLE
Assignment 1: Birthday Card || Assignment 2: Any own layout app

### PR DESCRIPTION
**Pull request by Priyanker**
Also Updated Journal for 3/6/2017

### CODE REVIEW

File: Priyanker/Assignment No 2/Project2.xml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        android:paddingTop="2dp"
+        android:text="@string/moc"
+        android:textAlignment="center"
+        android:textColor="?attr/colorPrimaryDark"
+        android:textSize="20sp"
+        android:textStyle="bold" />
+    <LinearLayout
+        android:id="@+id/mocweb"
+        android:background="#4eb980"
+        android:layout_width="wrap_content"
```
Comment (mallelapavank) [inline Priyanker/Assignment No 2/Project2.xml]: width should be match_parent here. To understand why, enter a short string in the website field like 15 characters or so and see what happens with both wrap_content and match_parent

Comment (Priyanker) [reply Priyanker/Assignment No 2/Project2.xml]: OH yes! typed a long text. So I added wrap_content. Didn't notice that. 
Will change it.

File: Priyanker/Assignment No 2/Project2.xml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        android:fontFamily="sans-serif"
+        android:layout_alignParentTop="true"/>
+    <ImageView
+        android:id="@+id/symbol"
+        android:layout_width="39.3dp"
+        android:layout_height="39.9dp"
+        android:layout_toRightOf="@id/title"
+        android:src="@drawable/fccsymbol"
+        android:layout_alignParentTop="true" />
+    <TextView
```
Comment (mallelapavank) [inline Priyanker/Assignment No 2/Project2.xml:26]: This extra bit of TextView is not necessary. What you can do instead is add alignParentEnd and marginEnd to the ImageView above. And make the title TextView width match_parent. By doing so your layout will be consistent on different screen sizes. Although what you have done is not wrong but this is just a better way to do it. 
```
        android:layout_marginEnd="30dp"
        android:layout_alignParentEnd="true"
```

Comment (Priyanker) [reply Priyanker/Assignment No 2/Project2.xml:26]: ya. I was having a bit of a problem with that image. Even a 0.1 dp extra is moving it away from the original height and leaving a gap. I tried so many ways but this is the best result it showed.

**Pull request merged**